crates/bindings/c/src/lib.rs

```diff
@@ -3,11 +3,26 @@
 
 mod types;
 
+use std::{sync::{Once, Mutex}, borrow::BorrowMut};
+
+use tokio::runtime::Runtime;
 use types::{
     blob, libsql_connection, libsql_connection_t, libsql_database, libsql_database_t, libsql_row,
     libsql_row_t, libsql_rows, libsql_rows_future, libsql_rows_future_t, libsql_rows_t,
 };
 
+static mut RT: Option<Mutex<Runtime>> = None;
```

Comment:
Is Mutex needed here at all? It looks like you only ever call `block_on` on the runtime, and it takes `&self`, so doesn't need mutual exclusion.

Reply:
I'm not sure if the Mutex is not helping in making sure those methods can be called from different threads. Will check.

Reply:
It seems that Runtime is Sync anyway so I will drop the Mutex.